### DISCUSSION TITLE
Best method to load non-geotagged image layers?

### DISCUSSION THREAD

**Discussion by robspearman** [pelicanmapping/osgearth#2307] [Support Q&A]
If I try to load a plain image as a GDALImage layer I get the error:  Resource unavailable: Dataset has no spatial reference information.

This happens even when I specify a profile, which used to be sufficient in osgEarth 2.

```
<Map name="Demo">
    <options>
        <terrain>
            <cluster_culling>false</cluster_culling>
            <skirt_ratio>0.0</skirt_ratio>
        </terrain>
    </options>
   
    <GDALImage name="test">
      <url>earth.jpg</url>
      <profile srs="+proj=latlong +a=6378140 +b=6378140" xmax="180" xmin="-180" ymax="90" ymin="-90" />
    </GDALImage>     

</Map>

```
What is the simplest way to do this in 3.4?

**Comment by gwaldron** [comment]
Best solution is to use the `gdal_translate` tool to georeference the file:
```
gdal_translate -a_srs epsg:4326 -a_ullr -180 90 180 -90 earth.jpg earth_geo.jpg
```

**Reply by robspearman** [reply]
I don't have write access to the images.  Is there some way to automate building a VRT file for any image given a known profile?  gdalbuildvrt won't work on non-geotagged images either.
